src/site/xdoc/server/quick-start.xml

```diff
@@ -36,7 +36,7 @@
 Step 0: Requirements
 ####################
 
-  * JRE 1.8
+  * JRE 1.11
```

Comment:
As far as I know there is no more JRE after Java 10, so you should just name it `JDK 11`